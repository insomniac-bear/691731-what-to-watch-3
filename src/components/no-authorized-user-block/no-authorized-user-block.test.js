import React from 'react';
import renderer from 'react-test-renderer';
import NoAuthorizedUserBlock from './no-authorized-user-block.jsx';

it(`Render NoAuthorizedUserBlock`, () => {
  const tree = renderer
    .create(
        <NoAuthorizedUserBlock
          activePageHandle={jest.fn()}
        />
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
