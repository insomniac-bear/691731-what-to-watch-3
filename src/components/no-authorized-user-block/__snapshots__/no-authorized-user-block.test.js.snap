// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render NoAuthorizedUserBlock 1`] = `
<div
  className="user-block"
>
  <a
    className="user-block__link"
    href="sign-in.html"
  >
    Sign in
  </a>
</div>
`;
